doc/md/compiler-ref.md

```diff
@@ -54,7 +48,6 @@ You can use the following options with the `moc` command.
 | `--stable-regions`                        | Force eager initialization of stable regions metadata (for testing purposes); consumes between 386KiB or 8MiB of additional physical stable memory, depending on current use of ExperimentalStableMemory. |
 | `--stable-types`                          | Compile binary and emit signature of stable types to `.most` file.                                                                                    |
 | `--stable-compatible <pre> <post>`        | Test upgrade compatibility between stable-type signatures `<pre>` and `<post>`.                                                                       |
-| `--rts-stack-pages <n>`                   | Set maximum number of pages available for runtime system stack (default 32).
```

Comment:
Is this no longer supported because we can't increase?

Reply:
Yes, this is a downside. We need to bound the Rust call stack size such that it does not collide with the persistent data in main memory.

Reply:
I increased the stack size to 4MB compared to the default 2MB of classical persistence. I believe the most stack-consuming process is the Candid destabilization when migrating from classical persistence to enhanced orthogonal persistence. Maybe we should consider using a separate temporary stack for this specific purpose, redirected to a heap-allocated blob...

Reply:
(Resolving the comment, as we have a second one on this topic below.)